tests/typecheck/fields/test_related.yml

```diff
@@ -642,8 +642,8 @@
         reveal_type(user.article_set) # N: Revealed type is "myapp.models.Article_RelatedManager"
         reveal_type(user.book_set.add)  # N: Revealed type is "def (*objs: Union[myapp.models.Book, builtins.int], *, bulk: builtins.bool =)"
         reveal_type(user.article_set.add)  # N: Revealed type is "def (*objs: Union[myapp.models.Article, builtins.int], *, bulk: builtins.bool =)"
-        reveal_type(user.book_set.filter)  # N: Revealed type is "def (*args: Any, **kwargs: Any) -> myapp.models.LibraryEntityQuerySet[myapp.models.Book]"
-        reveal_type(user.article_set.filter)  # N: Revealed type is "def (*args: Any, **kwargs: Any) -> myapp.models.LibraryEntityQuerySet[myapp.models.Article]"
+        reveal_type(user.book_set.filter)  # N: Revealed type is "def (*args: Any, **kwargs: Any) -> myapp.models.LibraryEntityQuerySet"
```

Comment:
Is it correct?

Reply:
Yeah, I think this is a chain reaction of a bug that was revealed. Might be better to push it as a separate PR though.

The below change is the reason, in `mypy_django_plugin/transformers/managers.py`

```diff
- manager_type_info = manager_instance.type
+ manager_type_info = manager_instance.type.get_containing_type_info(method_name)
```

Reply:
Right, think I figured it out. The change there takes a more correct path. There's nothing generic on the related managers, their model arguments are already populated on parent class(es). It becomes more obvious when using `.get` and seeing what model is returned. So I'll add that.

Reply:
I've opened #1701